music-playlist-app/index.js

```diff
@@ -83,8 +90,19 @@ function displaySong(song = "", artist = "") {
 }
 
 function playPauseSong() {
-  isPlaying ? song.pause() : song.play();
-
+  if (isPlaying) {
```

Comment:
The last thing I noticed is that, even after deleting all songs, it's still possible to play a song.

What might be good is to do an early return if the length of `playlist` is 0:

```suggestion
  if (playlist.length === 0) return;
  
  if (isPlaying) {
```

Early returns could also be added to `goToNextSong` and `goToPreviousSong`, which play a song after changing tracks.